Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/concepts/ConceptEditView.jsp

```diff
@@ -4,248 +4,248 @@
+                                + '\')">Remove</a>'
+                                + '</font></td>';
+                        htmlValue += '<td align="justify"><font size="2">'
+                                + eachSynonym.Word + '</font></td>';
+                        htmlValue += '<td align="justify"><font size="2">'
+                                + eachSynonym.Description + '</font></td></tr>';
+                        $('#addedSynonymsTable tr:last').after(htmlValue);
 
-					}
-				});
-	};
+                    }
+                });
+    };
 
-	$(document)
-			.ready(
-					function() {
-						$('#synonyms').dataTable({
-							"bJQueryUI" : true,
-							"sPaginationType" : "full_numbers",
-							"bAutoWidth" : false
-						});
-						var conceptid = $('#conceptid').val();
-						$
-								.ajax({
-									type : "GET",
-									url : "${pageContext.servletContext.contextPath}/getConceptEditSynonyms",
-									data : {
-										conceptid : conceptid
-									},
-									success : function(response) {
-										var border = response.length > 0 ? 1
-												: 0;
-										var html = '<table border="'+ border +'" width="400" id="addedSynonymsTable"><thead></thead><tbody>';
-										var len = response.length;
-										for (var i = 0; i < len; i++) {
-											html += '<tr><td align="justify"><font size="2">'
-													+ '<a onclick="synonymRemove(\''
-													+ response[i].id
-													+ '\')">Remove</a>'
-													+ '</font></td>';
-											html += '<td align="justify"><font size="2">'
-													+ response[i].word
-													+ '</font></td>';
-											html += '<td align="justify"><font size="2">'
-													+ response[i].description
-													+ '</font></td></tr>';
-										}
-										html += '</tbody></table>';
-										$("#addedSynonyms").html(html);
+    $(document)
+            .ready(
+                    function() {
+                        $('#synonyms').dataTable({
+                            "bJQueryUI" : true,
+                            "sPaginationType" : "full_numbers",
+                            "bAutoWidth" : false
+                        });
+                        var conceptid = $('#conceptid').val();
+                        $
+                                .ajax({
+                                    type : "GET",
+                                    url : "${pageContext.servletContext.contextPath}/getConceptEditSynonyms",
+                                    data : {
+                                        conceptid : conceptid
+                                    },
+                                    success : function(response) {
+                                        var border = response.length > 0 ? 1
+                                                : 0;
+                                        var html = '<table border="'+ border +'" width="400" id="addedSynonymsTable"><thead></thead><tbody>';

```

Comment:
I think I would put the table into the base html, and then just add/append the rows.